signing_service/signing_service/signing_service.py

```diff
@@ -341,6 +337,10 @@ def _was_sigterm_caught(self) -> bool:
         """ Helper function which checks if SIGTERM signal was caught. """
         return self.was_sigterm_caught
 
+    def _set_was_sigterm_caught_true(self, signum: int, frame: Optional[FrameType]) -> None:  # pylint: disable=unused-argument
```

Comment:
Why `signum: int` and `frame: Optional[FrameType]` are in function declaration?

Reply:
This function is connected to signal. It means that when signal is being send, this function will be called with arguments send by signal. And this function is called with arguments send by `signal.SIGTERM`. They come from standard library code. We don't do anything with them, but we still have to accept them.

Reply:
thanks for explanation